Sources/MapboxNavigation/OverlayController.swift

```diff
@@ -0,0 +1,390 @@
+
+import UIKit
+import MapboxDirections
+import MapboxCoreNavigation
+import MapboxMaps
+import Turf
+import CoreLocation
+
+
+class OverlayController: NavigationComponent, ViewCycleObserver {
```

Comment:
The term “overlay” often refers to a kind of layer within the map view. The map SDK and this library tend to use the term “ornament” instead.

Reply:
Done